FRAuth/FRAuth/Authentication/Node.swift

```diff
@@ -203,7 +211,7 @@ public class Node: NSObject {
                             }
                             else {
                                 if let token = accessToken {
-                                    try? self.sessionManager?.setAccessToken(token: token)
+                                    try? self.keychainManager?.setAccessToken(token: token)
```

Comment:
Super minor, but you could make it "let _ = try? self.keychainManager?.setAccessToken(token: token)" to get read of the warning

Reply:
Yes, I've updated all compiler warnings to correctly handle the exception from the try/catch block. We should try to catch most of exceptions to provide more information to the developers. It wasn't really minor, but was really important catch. Thanks.